alchemy/test/cloudflare/secrets-store.test.ts

```diff
@@ -0,0 +1,173 @@
+import { describe, expect } from "vitest";
+import { alchemy } from "../../src/alchemy.ts";
+import { secret } from "../../src/secret.ts";
+import { createCloudflareApi } from "../../src/cloudflare/api.ts";
+import { SecretsStore } from "../../src/cloudflare/secrets-store.ts";
+import { BRANCH_PREFIX } from "../util.ts";
+
+import "../../src/test/vitest.ts";
+
+const test = alchemy.test(import.meta, {
+  prefix: BRANCH_PREFIX,
+});
+
+describe("SecretsStore Resource", () => {
```

Comment:
Please add tests to check that additional secrets can be added or removed to test that secrets are automatically added and removed from the store.